scripts/build/postbuild.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const pkgPath = `${process.cwd()}/package.json`;
+const pkgPathTmp = `${process.cwd()}/.tmp.package.json`;
+
+fs.copyFileSync(pkgPathTmp, pkgPath);
```

Comment:
file ops can always fail, since you're not the exclusive owner of the file system - guard against failure here and elsewhere